### DISCUSSION TITLE
What projects have you seen that use Inter?

### DISCUSSION THREAD

**Discussion by applecuckoo** [rsms/inter#492] [General]
Hey there all, I was checking out things and saw that the NZ government department [LINZ](https://www.linz.govt.nz/) recently launched a new website that uses Inter as a secondary font. Unfortunately, it's still relying on the old, dusty Google Fonts CDN instead of the 'official' CDN on rsms.me. What other projects have you seen with Inter used?

**Comment by applecuckoo** [comment]
This is interesting - the ISO seems to be using a custom version of Inter on [their website](https://www.iso.org/home.html), plus they seem to be using a variable version and not a static one.

**Reply by applecuckoo** [reply]
*previous comment deleted, image cleaned up due to privacy, here it is again

> Looks like normal Roboto to me

Really? This is what it looks on my browser, the black box is the WhatFont plugin I use on Firefox to check fonts (sometimes I use developer tools too)

![ISO - International Organization for Standardization — Mozilla Firefox_001](https://user-images.githubusercontent.com/113647417/215197106-b5fa3949-4f3f-46b8-a96b-53975561e63d.png)

**Comment by lxgw** [comment]
GitHub client app on Android.

**Comment by quiple** [comment]
In Korea, [Pretendard](https://github.com/orioncactus/pretendard) the derived version of Inter is often used instead of Inter. It is widely used in many places, including national institutions, corporations, and organizations. Some examples here: [The Korean Committee for UNICEF](https://www.unicef.or.kr/event/unicef-team3-1/), [Cheong Wa Dae: Back to the people](https://opencheongwadae.kr/)

**Reply by kenmcd** [reply]
> How do you know Pretendard is a derived version from Inter?

It says Inter is the Base here (near bottom): 
https://github.com/orioncactus/pretendard/blob/main/packages/pretendard/docs/en/README.md

**Comment by alexleduc76** [comment]
I forgot which one it was, but there is a Linux distro the uses it as the UI font. I use Manjaro Linux and I have changed the UI font to Inter Light. You can tell this font was made for UIs
![Screenshot_20230127_152849](https://user-images.githubusercontent.com/13103140/215190852-ad95af97-29c5-4612-b90a-798e4ffaf19d.png)

**Reply by kepstin** [reply]
[elementary OS](https://elementary.io/) does as well - both on their website, and as the standard UI font, which you can see in their screenshots.

**Comment by mon-jai** [comment]
GitLab is now [using a rebanded version of Inter as their main UI font](https://about.gitlab.com/blog/2023/01/17/new-typefaces-in-gitlab/).

**Reply by rsms** [reply]
> GitLab is now [using a rebanded version of Inter as their main UI font](https://about.gitlab.com/blog/2023/01/17/new-typefaces-in-gitlab/).

There's a huge (~40 meter) freeway GitLab ad near where I live with Inter on it. Puts a smile on my face whenever I drive by it :•)

**Reply by kepstin** [reply]
Reading through the [📣 Feedback issue: New fonts in GitLab](https://gitlab.com/gitlab-org/gitlab/-/issues/386205) is an interesting read, since it has discussion about some of the issues they've run into and user feedback on the font design which might be applicable to upstream Inter. (Although, be aware that a lot of comments there are talking the monospace font choice.)

It is really cool that they're using a version of the Inter variable font with optical size axis enabled, and browsers are automatically using that to adjust heading vs. body text.

The choice to enable the disambiguation glyph designs by default is kind of interesting, and totally understandable when used in a context where it's displaying user-defined strings (like git repo names) where they need to make sure to avoid confusable characters. I wonder if they still have the `zero` feature available in their modified font so they can select to use slashed zero in specific locations where a `0`/`O` confusion is a problem (like Twitter does on `@` handles).

It's not mentioned in that blog post, but they've also disabled the `calt` feature. I personally find `calt` to be distracting when typing (and you type a lot in the Gitlab UI!), since it causes characters to jump around, and it's kind of confusing since the same character can have multiple different appearances within a single sentence/line.


I actually apply some similar tweaks when I use Inter as a UI font (using fontconfig on linux rather than with a customized font), although I personally use `ss04` instead of `cv05` (I like the big `I`), and I also enable `tnum` by default, since a lot of desktop apps don't enable `tnum` in places where it's useful, like when showing lists of filenames.

**Reply by rsms** [reply]
> Reading through the [📣 Feedback issue: New fonts in GitLab](https://gitlab.com/gitlab-org/gitlab/-/issues/386205) is an interesting read, since it has discussion about some of the issues they've run into and user feedback on the font design which might be applicable to upstream Inter. (Although, be aware that a lot of comments there are talking the monospace font choice.)

Very interesting. Thank you for sharing this.

> The choice to enable the disambiguation glyph designs by default is kind of interesting

From what I can tell, they only enabled a few alternate glyphs by default, but not the whole set. For example, the regular simple capital i ("I") is used while the alternate lower-case L ("l") is used. Makes sense to me. @twardoch's excellent [OpenType Feature Freezer](https://twardoch.github.io/fonttools-opentype-feature-freezer/) (formerly pyftfeatfreeze) makes it a breeze to customize the "defaults" of fonts.

> It's not mentioned in that blog post, but they've also disabled the `calt` feature. I personally find `calt` to be distracting when typing (and you type a lot in the Gitlab UI!), since it causes characters to jump around, and it's kind of confusing since the same character can have multiple different appearances within a single sentence/line.

This is probably the most interesting to me, to learn more about. calt really should not contain stuff that feels like an "extra optional feature" (and of course should not be distracting.)

@kepstin What is a specific example of "I personally find `calt` to be distracting when typing"? Is is only the parenthesis (which I think might be "too clever" and I might remove that in version 4.)

**Reply by kepstin** [reply]
> @kepstin What is a specific example of "I personally find `calt` to be distracting when typing"? Is is only the parenthesis (which I think might be "too clever" and I might remove that in version 4.)

The parenthesis/brackets are distracting, yes. But the characters that I find most problematic are the vertical movement of `:`, `*`,  `÷`, `-`, and `x` (letter x) when used between numbers, like 12:30 7*4  7×4  7÷8 8-7.

In particular, moving the x vertically is a glyph confusion issue, since that means it might be confused with `×` (multiplication sign). This is most often a problem with numbers written in hex like 0x40392817, where the x needs to specifically be a letter.

The movement of `÷` is kind of odd - why isn't the default vertical alignment of that suitable for use with numbers? It shouldn't have to move.

Also, the vertical realignment of characters when typing certain specific combinations like `:-)` (Yes, i know that's a smiley, but most people use emoji instead nowadays, and it still looks fine without the vertical movement.)

**Comment by ghost** [comment]
macOS App CodeEdit, on its website, uses Inter as an alternative to SF Pro.

[Website](https://codeedit.app)

**Comment by michael-andreuzza** [comment]
I am using Inter on [lexingtonthemes.com](https://lexingtonthemes.com/) 
![lexingtonthemes com e7234ebe](https://github.com/rsms/inter/assets/34077176/b2b8ba17-0974-4ea1-9468-2b4624e7bbd7)

**Comment by vitorsr** [comment]
![](https://github.com/rsms/inter/assets/40741601/a53b570b-7712-476b-8469-2363301b7518)

![](https://github.com/rsms/inter/assets/40741601/0ec93a4f-c19b-44ea-806a-3162bf7535e1)

![](https://github.com/rsms/inter/assets/40741601/57b6bd21-27d1-4df8-8e5e-599da54e9b04)

![](https://github.com/rsms/inter/assets/40741601/b5983cbd-8acf-4383-a534-788d9380d3d5)

![](https://github.com/rsms/inter/assets/40741601/84065375-8050-4435-b6d0-67958d8a43a6)

![](https://github.com/rsms/inter/assets/40741601/12a80130-8f54-4727-9cf5-70f686da83dd)

![](https://github.com/rsms/inter/assets/40741601/e5bc6b90-5414-4514-8a77-8e60e640fe01)

![](https://github.com/rsms/inter/assets/40741601/fdb0c310-9b1d-45cc-8b6f-35a92b9fd5db)

![](https://github.com/rsms/inter/assets/40741601/5b98bd09-490e-4c29-87ca-5042411692e2)

![](https://github.com/rsms/inter/assets/40741601/db88119f-d298-481b-bb48-e55be939a1d2)

**Comment by maa7** [comment]
https://invisioncommunity.com/

**Comment by vitorsr** [comment]


**Reply by ghost** [reply]
That looks like 15 different fonts in one. Where is the Inter?

**Comment by frnzkng** [comment]
Erste Sparkasse (An Austrian bank) also uses Inter on their website and on their real posters
![chrome_yChhDWzD2L](https://github.com/rsms/inter/assets/136699979/2c7c1300-4443-4795-beb8-4969684646b9)

**Comment by applecuckoo** [comment]
As of v4.0, Blender now uses Inter as its UI font instead of DejaVu Sans:
![interinblender](https://github.com/rsms/inter/assets/113647417/19b0c7e1-b91c-486d-897b-6363f85db3f9)
They've even made a big deal about it in the [release notes](https://www.blender.org/download/releases/4-0/#font).

**Comment by mazmap** [comment]
https://literal.club: 

![Capture-2024-03-09-011018](https://github.com/rsms/inter/assets/46203814/0bdaedf7-b947-400a-b3a6-788c1d734618)

**Comment by applecuckoo** [comment]
A major New Zealand news site, the New Zealand Herald, just [switched to Inter](https://www.nzherald.co.nz/nz/masthead-and-typeface-the-start-of-full-new-zealand-herald-website-and-app-redesign/ATMKN2VJ2JEDDNJ6ALOVVSFOPI/) as part of a redesign. Here's a bit of info from the article:
> For type fans, our new font is Inter. It is widely used because of its big x-height and readability. It was created by designer [Rasmus Andersson](https://rsms.me/about/). We will continue to use our slab serif, ***Stag***, for Herald brand messaging.

![Screenshot 2024-04-09 at 14-55-11 NZ Herald Latest NZ news plus World Sport Business and more - NZ Herald](https://github.com/rsms/inter/assets/113647417/2bba7f39-357a-416a-8862-bee1c263028b)

**Comment by iberdinsky-skilld** [comment]
New Drupal Navigation module
https://www.drupal.org/project/drupal/issues/3438895

It is still experimental. But probably soon became stable part of Drupal Core functional.

**Comment by alexleduc76** [comment]
It's not decided yet, but the Gnome desktop environment is considering changing its default UI font from Cantarell to Inter. Considering Gnome is one of the most used DEs on Linux, that would mean most Linux distributions would ship with this font as default.
https://youtu.be/KzXl6a8iOfw?si=2j5-EEl8VLXFQLW-&t=376

**Comment by andergisomon** [comment]
<img width="1470" height="770" alt="Screenshot 2026-06-14 at 6 41 38 PM" src="https://github.com/user-attachments/assets/b26b30f3-3efc-4f16-a08a-91f145ca4ec7" />

The Malaysian Department of Statistics uses Inter for one website of theirs, OpenDOSM.
